Workbooks/PostgreSQL Flexible Server/Performance/High Temporary Files/High Temporary Files.workbook

```diff
@@ -2649,29 +2727,25 @@
+        "query": "let graphgrain=iff({TimeRange:grain}<15m,15m, {TimeRange:grain});\r\nlet TimeStart = todatetime('{GlobalStartTime}');\r\nlet TimeEnd = todatetime('{GlobalEndTime}');\r\nlet databasename = \r\n{SessionsTable}\r\n| distinct DatabaseName,DatabaseId;\r\n{QueryStoreTable}\r\n| where UserId != 10   \r\n| where QueryId == {QueryIdsByTemp}\r\n| distinct MeanExecDurationMs,MaxExecDurationMs,MinExecDurationMs,Calls,QueryId,QueryType,Rows,TotalExecDurationMs,StartTime,EndTime,PlanId,DatabaseId,UserId,TempBlksWritten,TempBlksRead, RuntimeStatsEntryId \r\n| where todatetime(EndTime) >= TimeStart and todatetime(EndTime) <= TimeEnd\r\n| extend total_Temp_blks_written = TempBlksWritten*8\r\n| extend total_Temp_blks_read = TempBlksRead*8\r\n| extend Mean_Temp_blks_written=TempBlksWritten*8/Calls\r\n| extend Mean_Temp_blks_read=TempBlksRead*8/Calls\r\n| summarize WindowStart=min(StartTime), WindowEnd=max(EndTime), MeanTempBlksWritten=avg(Mean_Temp_blks_written),MeanTempBlksRead=avg(Mean_Temp_blks_read),MinTempBlksWritten=min(Mean_Temp_blks_written), MaxTempBlksWritten=max(Mean_Temp_blks_written),MeanRows=sum(Rows)/sum(Calls),TotalRows=sum(Rows),TotalCalls=sum(Calls), MeanExecutionTime=avg(MeanExecDurationMs),MinExecutionTime=min(MinExecDurationMs),MaxExecutionTime=max(MaxExecDurationMs),TotalExecutionTime=sum(TotalExecDurationMs),TotalTempBlksWritten=sum(total_Temp_blks_written) by bin(todatetime(EndTime), graphgrain), QueryId,QueryType,PlanId,DatabaseId,UserId\r\n| join kind=leftouter databasename on $left.DatabaseId == $right.DatabaseId\r\n| project QueryId,QueryType,PlanId,DatabaseName = iff(isempty(DatabaseName),strcat(\"oid::\",DatabaseId), DatabaseName), UserId, WindowStart, WindowEnd,MeanRows, MeanTempBlksWritten, MeanTempBlksRead,MeanExecutionTime, TotalRows,TotalTempBlksWritten, TotalCalls, TotalExecutionTime, MinTempBlksWritten, MaxTempBlksWritten, MinExecutionTime, MaxExecutionTime\r\n|order by WindowStart asc",
         "size": 0,
-        "title": "QueryID {QueryIdsByTemp} runtime statistics split by time buckets of {ChartGrain} each",
+        "title": "QueryId {QueryIdsByTemp} Runtime Statistics Split by Time Buckets of {ChartGrain} Each",
         "noDataMessageStyle": 4,
         "showExportToExcel": true,
         "queryType": 0,
-        "resourceType": "microsoft.dbforpostgresql/flexibleservers",
+        "resourceType": "microsoft.resources/resourcegroups",
```

Comment:
Why you selected the RG? 

Can you double check if this is not introducing a regression?

Reply:
Corrected 


![image](https://user-images.githubusercontent.com/127991103/236349433-1350823f-543c-40a3-82bd-556337177b2f.png)